### ISSUE TITLE
Key signatures duplicate after applying Insert one measure before selection command several times

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#22324]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Steps to reproduce

1. Create score
2. Add key signatures
3. Select first measure
4.  Apply **Insert one measure before selection** command more than 10 times (press **Insert** on Windows)

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/90187801/a58d6d0c-066b-439d-b466-e04780417e37

### MuseScore Version

4.4

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

all

**Comment by RobFog** [comment]
If this is a regression from #22279 which fixed issues targeted for MuseScore 4.3 wouldn't the MuseScore version for this bug be 4.3 instead 4.4?

**Comment by DmitryArefiev** [comment]
Nope. It's not a common case. If user wants to insert many bars he probably will be use Insert measure dialog
